### DISCUSSION TITLE
Contribution Graph doesn't show all contribution

### DISCUSSION THREAD

**Discussion by yvzksgl** [community/community#86493] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Github documentation says:

>  Commits will appear on your contributions graph if they meet all of the following conditions:
> 
> - The email address used for the commits is associated with your account on GitHub.com.
> - The commits were made in a standalone repository, not a fork.
> 
> The commits were made:
> 
> - In the repository's default branch
> - In the gh-pages branch (for repositories with project sites)

What is the reason of it. I'm generally working on branches, then creating PR, when the team convinced the added feature is realiable and necessary. It makes my contribution grapf 1/4 of it should be. I'm using gitkraken by the way but as far as i understand it's not about gitkraken.

**Comment by knownothing10** [comment]
GitHub has specific criteria for displaying commits on your contributions graph, which includes the conditions you mentioned:

The email address used for the commits is associated with your account on GitHub.com: This requirement ensures that the commits are linked to your GitHub profile. You need to use an email address that is registered and connected to your GitHub account.

The commits were made in a standalone repository, not a fork: Forked repositories have separate contribution graphs, as they are considered separate repositories. Commits made in a forked repository would contribute to the original repository's contribution graph, not to your personal graph.

The commits were made in the repository's default branch or the gh-pages branch (for repositories with project sites): The contributions graph mainly focuses on the default branch of a repository or the gh-pages branch for repositories with GitHub Pages enabled. This is because these branches typically represent the main development and deployment branches.

If you are primarily working on branches and creating pull requests, it's important to note that the contributions graph emphasizes work completed on the default branch or the gh-pages branch. However, the contributions on other branches are still recognized and linked to the respective pull requests.

While using GitKraken to interact with Git doesn't directly impact the visibility of your contributions on the graph, it's essential to ensure that your commits meet the criteria mentioned above to appear on your contributions graph accurately.

Remember that the contributions graph is just one way to visualize your contributions on GitHub. Pull requests and code reviews are valued contributions to a project, even if they don't directly impact your personal contributions graph. The primary goal is to collaborate effectively and deliver reliable features to the team.

I hope this helps clarify the reasons behind the criteria. Let me know if there's anything else I can assist you with!
